avocado_vt/plugins/vt_resolver.py

```diff
@@ -41,9 +41,11 @@ def _get_reference_resolution(self, reference):
         runnables = [self._parameters_to_runnable(d) for d in
                      cartesian_parser.get_dicts()]
         if runnables:
-            if self.config.get("nrunner.max_parallel_tasks", 1) != 1:
-                warnings.warn("The VT NextRunner can be run only with "
-                              "nrunner-max-parallel-tasks set to 1")
+            if self.config.get(
+                "run.max_parallel_tasks", self.config.get(
+                    "nrunner.max_parallel_tasks", 1)) != 1:
+                warnings.warn("The VT NextRunner can be run only "
+                              "with max-parallel-tasks set to 1")
```

Comment:
@luckyh @richtja This might seem like a random though but as we already run Avocado VT tests in parallel if you are both interested I can assist you in this repo to make this possible for everyone using the LXC Spawner from https://github.com/avocado-framework/avocado/pull/4158. @luckyh One particular advantage of using an LXC container isolation here is that compared to distributed computing it could allow for QCOW2 image symlinks and other same-filesystem caching that will reduce the need of copying/downloading/uploading of these in a cluster. Of course, resources eventually will reach some limit on the same machine and would need distributed extension - both solutions complement each other in the end.